src/scss/base/_mixins.scss

```
@@ -0,0 +1,76 @@
+@mixin marginX($number) {
+   margin: 0 $number;
+}
+
+@mixin marginY($number) {
+   margin:  $number 0;
+}
+
+@mixin marginA($number) {
+    margin: $number;
+}
+
+@mixin paddingX($number) {
+   padding-left: $number;
+   padding-right: $number;
+}
+
+@mixin paddingY($number) {
+   padding-top:  $number;
+   padding-bottom:  $number;
+}
+
+@mixin paddingA($number) {
+    padding: $number;
+}
+
+@mixin position($prop, $top:null, $left:null, $right:null, $bottom:null){
+    position: $prop;
+    top: $top;
+    left: $left;
+    bottom: $bottom;
+    right: $right;
+}
+
+@mixin textStyle($fontsize, $textColor, $fontweight, $transform){
+    font-size: $fontsize;
+    color: $textColor;
+    font-weight: $fontweight;
+    text-transform: $transform;
+}
+
+@mixin cssPrefix($prop,$value) {
+    -webkit-#{$prop}: $value;
+    -moz-#{$prop}: $value;
+    -ms-#{$prop}: $value;
+    -o-#{$prop}: $value;
+    #{$prop}: $value; 
+}
+
+@mixin border($width, $color, $direction) {
+    @if($direction == all){
+        border: $width solid $color;
+    }@else{
+        border-#{$direction}: $width solid $color;
+    }
+}
+
+@mixin button($bgColor, $colorText, $textTransform) {
+    background-color:$bgColor;
+    color:$colorText;
+    text-transform: $textTransform;
+}
+
+.button{
```

```suggestion
.button {
```
Sửa hết những thứ tương tự